src/Components/InstanceTypesSelect/index.js

```diff
@@ -106,26 +110,37 @@ const InstanceTypesSelect = ({ setValidation, architecture }) => {
   const types = filteredTypes || instanceTypes;
 
   return (
-    <Select
-      ouiaId="select_instance_type"
-      variant="typeahead"
-      aria-label="Select instance type"
-      placeholderText="Select instance type"
-      maxHeight="450px"
-      isOpen={isOpen}
-      selections={wizardContext.chosenInstanceType}
-      onToggle={onToggle}
-      onSelect={onSelect}
-      onFilter={onFilter}
-      {...(numOptions < types.length && {
-        loadingVariant: {
-          text: `View more (${types.length - numOptions})`,
-          onClick: () => setNumOptions(numOptions + OPTIONS_PER_SCREEN),
-        },
-      })}
-    >
-      {selectItemsMapper(types, numOptions)}
-    </Select>
+    <>
+      {!isTypeSupport && (
+        <Alert
+          data-testid="unsupported_type_alert"
+          ouiaId="instance_type_not_supported_alert"
+          variant="warning"
+          isInline
+          title="Warning: The selected type is not sufficient for the chosen image"
```

Comment:
```suggestion
          title="Warning: The selected type does not  have sufficient resources for the chosen image"
```

Reply:
@lzap suggested the following warning:
`The selected specification does not meet minimum requirements for this image. Launch will fail.`

Reply:
All I drop into comments are just ideas, make your own pick. Or if you don’t like it, that is fine.

Maybe "Launch will fail" is too strict, so only the first sentence? Or I like the fixed version (@ezr-ondrej proposal) too.